crawl-ref/source/dat/des/branches/swamp.des

```diff
@@ -2856,6 +3050,65 @@ Ztt..Kttttt..ttZ
 ZtrrrrtZZtqqqqtZ
 ENDMAP
 
+NAME:    nicolae_swamp_thats_no_hydra
+TAGS:    transparent no_pool_fixup
+DEPTH:   Swamp
+KMONS:   1 = kraken
+KFEAT:   @ = . / W
+KFEAT:   1 = w
+SHUFFLE: LMN, OPQ, RSZ
+NSUBST:  `- = 2=H / 2=H. / .
+SUBST:   LOR = @, MNPQSZ = t, - = .W, ` = wW, D = tww
+: dgn.delayed_decay(_G, 'H', 'hydra skeleton')
+MAP
+     LLLLM
+   ZZLLLLMMM
+  ZZZ-----MMM
+ ZZ--`````--MM
+ ZZ-`DDDDD`-NN
+SS-`DDDDDDD`-NN
+SS-`DDwwwDD`-NN
+SS-`DDw1wDD`-NN
+SS-`DDwwwDD`-OO
+RR-`DDDDDDD`-OO
+ RR-`DDDDD`-OO
+ RR--`````--OO
+  RRQ-----PPP
+   RQQQQPPPP
+     QQQPP
+ENDMAP
+
+NAME:    nicolae_swamp_vermiculated
+TAGS:    transparent patrolling
```

Comment:
Feels like this vault probably doesn't need patrolling. For one it has a lot of walls that will block noise and prevent worms from wandering out so freely. Secondly it's all pretty standard Swamp fodder that's not going to be confusing to see outside of the vault. And finally it's probably good to break up the swamp worms so they're not so penned in anyhow, since they need LOF to do their thing. Generally think it's better to avoid use of patrolling unless there's a compelling use case like a theme that requires monsters to be in place for it to register, especially in more open vaults.